editions/empty/tiddlywiki.info

```diff
@@ -17,7 +17,7 @@
 		"static": [
 			"--rendertiddler","$:/core/templates/static.template.html","static.html","text/plain",
 			"--rendertiddler","$:/core/templates/alltiddlers.template.html","alltiddlers.html","text/plain",
-			"--rendertiddlers","[!is[system]]","$:/core/templates/static.tiddler.html","static","text/plain"],
-			"--rendertiddler","$:/core/templates/static.template.css","static/static.css","text/plain"
+			"--rendertiddlers","[!is[system]]","$:/core/templates/static.tiddler.html","static","text/plain",
+			"--rendertiddler","$:/core/templates/static.template.css","static/static.css","text/plain"]
```

Comment:
Yea, but the issue imo is a "copy/paste problem" because the `]` hasn't been in its own line.